tests/test_lib.py

```diff
@@ -35,6 +36,8 @@
 PAGE_URL = f"{PAGES_URL}/{PAGE_ID}"
 ANNOTATION_ID = 1863864
 ANNOTATION_URL = f"{ANNOTATIONS_URL}/{ANNOTATION_ID}"
+ORGANIZATION_ID = "1"
```

Comment:
It's good to follow the syntax that is already used in the script. This could probably be an integer.